tas-easy-install.sh

```diff
@@ -50,6 +50,30 @@ install_sso_keycloak() {
         exit 1
     fi
 }
+
+# Generate the script to initialize the environment variables for the service endpoints
+generate_env_script() {
+    # Write the script to a file
+cat <<EOL > tas-env-variables.sh
+#!/bin/bash
```

Comment:
instead of writing to a bash script, i think we need to write to an env file, then instruct the user to 
`source ./tas_env` in their terminal.
The env file would look like
```
export BASE_HOSTNAME=apps.$(oc get dns cluster -o jsonpath='{ .spec.baseDomain }')
export KEYCLOAK_REALM=sigstore
export FULCIO_URL=https://fulcio.$BASE_HOSTNAME
export KEYCLOAK_URL=https://keycloak-keycloak-system.$BASE_HOSTNAME
export REKOR_URL=https://rekor.$BASE_HOSTNAME
export TUF_URL=https://tuf.$BASE_HOSTNAME
export KEYCLOAK_OIDC_ISSUER=$KEYCLOAK_URL/auth/realms/$KEYCLOAK_REALM
```

Reply:
@omveda I see now you instruct the user to run `. ./tas-env-variable.sh`  and that `. ` has the same effect as `source tas-env-variable.sh` so this _does_ work - but we should clarify that a bit, otherwise most will run the script as ./tas-env-variable.sh and it won't work.